Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM  mhart/alpine-node:11.0.0
+
+# install dependencies
+RUN apk add git
+
```

Comment:
It would be cool to set NODE_ENV=production I guess.
Right now we don't use it, but we could use it to filter some log by default.

Reply:
Ok, I will do it ASAP,

Thank's for the review